vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -51,42 +51,41 @@ A customized image can include:
 * New APIs 
 * Legacy Entando plugins
 * New database tables  
-* Other extensions to the **entando-core** 
+* Other extensions to the entando-core
 
-It is recommended that most extensions to the Entando Platform occur in microservices. However, legacy 
+It is recommended that most extensions to the Entando Platform are made in the microservices. However, legacy 
```

Comment:
can you tighten this up to indicate who/what is doing the making? this either reads that 1) extensions are created (by a user) in the microservices or 2) extensions are executed in the microservices; the previous version is more accurate and precise

Reply:
switched back to the original but this sentence is about making customizations and occur is less accurate since it makes it sound like it's just something that happens, without anyone doing anything. And if the customization is created in the ms, then it has to be executed there, even if other components are involved.  

At any rate, edited and simplified.